ONSdigital/sdx-compose
# Add publisher

### Purpose of pull request

The purpose of this PR is to point the seft-publisher-service to check for files in EDC_Template_RAS, as well as to set the rate at which it checks this folder to 2.5 minutes.

### What has changed

Modified the ftp.env file to add new variables: 
* `SEFT_PUBLISHER_FTP_FOLDER=/EDC_Template_RAS/`
* `SEFT_FTP_INTERVAL_MS=150000`

### How to test

Run the `make build` command in your sdx-compose directory, followed by `make start`. The seft-publisher-service should look for files every 2.5 minutes.

### Links

[Trello card](https://trello.com/c/9oUzOLTk)
